tests/unit/roster/test_clustershell.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+'''
+unit tests for clustershell roster
+'''
+# Import Python libs
+from __future__ import absolute_import
+
+# Import Salt Testing libraries
+from tests.support.unit import TestCase, skipIf
+from tests.support.mock import (
+        NO_MOCK,
+        NO_MOCK_REASON,
+        MagicMock,
+        patch)
+
+
+@skipIf(NO_MOCK, NO_MOCK_REASON)
+class ClusterShellTestCase(TestCase):
```

Comment:
These tests should be gated if `ClusterShell` isn't available.

Reply:
@cachedout I have added this gating in https://github.com/cachedout/salt/pull/18. Can you merge that in here? I have also added a ClusterShell pip install to the salt-jenkins states in https://github.com/saltstack/salt-jenkins/pull/656.